user_manual/groupware/sync_ios.rst

```diff
@@ -39,6 +39,16 @@ You should now find your contacts in the address book of your iPhone.
 .. note:: Beginning with iOS 12 an SSL encryption is necessary. Therefore do **not** disable **SSL**
   (For this reason a certificate is required at your domain, https://letsencrypt.org/ will do).
 
+The usual link syntax looks like this:
+
+https://<fqdn-of-server>/remote.php/dav/calendars/<username>/<calendar-name>/
+
+iOS is a little bit special and usually requires the following syntax:
+
+https://<fqdn-of-server>/remote.php/dav/principals/users/<username>/
```

Comment:
![image](https://github.com/nextcloud/documentation/assets/1374172/c721ded5-9f5d-4d09-8459-a658933960b3)

The Calendar app knows and shows a general link and one for ios/macos

Reply:
It's old. Could you share a screenshot?